scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
+
+# Get the path to the script directory
+SCRIPT_DIR="$(eval dirname $0)"
+
+# Get the SHASUM of the tarball
+ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
+SHASUM="$(eval /usr/bin/shasum -a 256 ${ZIP_FILE})"
```

Comment:
Sorry for three separate comments on the same line!  I think including the `/usr/bin` might be less ideal here, but it's hard to say.  This wouldn't respect a users `PATH` though.

I'd probably say leave it for now, but this might break on my own computer.  Specifically, `shasum` is a **PERL SCRIPT** 😱, but the version of Perl that Apple ships is not the the same as the verison of Perl that my Homebrew has installed at the mercy of its own dependency needs.

That's to say, when I run `which shasum`, I and most things actually use `/usr/local/bin/shasum` which uses its matching version of Perl.

I guess I'm just saying the `/usr/bin` could be removed.

Reply:
but if you're using everyone's `/usr/bin/shasum` then you should be able to have a consistent value there, right?

Reply:
I was just generally highlighting that using explicit paths is sometimes less desirable since it doesn't respect a user's `PATH`.  For example, one might also consider using the full path to `tar`, `dirname`, `echo`, (etc.), but those aren't specified and I didn't see a specific reason for `shasum` being an exception.

My scenario is likely an extreme edge case, and possibly wouldn't break.  Anyhow, I'm certainly not objecting, just pointing out the subtle nuanced difference in file resolution patterns.

Reply:
I deliberately set it to `/usr/bin/shasum`, because that's the one shipped with the OS. `/usr/local/bin/shasum` often is a custom installation, or something from Homebrew, which might yield different output.

Reply:
Yeah I think explicitly using the system one here is probably the best way to handle it.